src/plugins/home/public/application/components/use_case_card.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { ContentManagementPluginStart } from '../../../../content_management/public';
+
+export const registerUseCaseCard = (
+  contentManagement: ContentManagementPluginStart,
+  workspaceEnabled: boolean,
+  {
+    target,
+    order,
+    id,
+    title,
+    description,
+    icon,
+  }: {
+    target: string;
+    order: number;
+    id: string;
+    title: string;
+    description: string;
+    icon: string;
+  }
+) => {
+  if (workspaceEnabled) return;
```

Comment:
Move `workspaceEnabled` check out to where `registerUseCaseCard` was called and do a if condition from there.

Reply:
Thanks for the comments, already updated.